### ISSUE TITLE
custom.css is ignored

### ISSUE THREAD

**Issue by Lorentz83** [bouncepaw/mycorrhiza#258]
according to https://mycorrhiza.wiki/help/en/file_structure a file in `static/custom.css` should be loaded after the main style allowing customization.

But this file, despite being served by the webserver, is not included in the html pages. 

A simple search in the whole codebase for `custom.css` returns only the documentation and the http handler, but the file name is not mentioned anywhere else near the templates.

```
$ rgrep custom.css
misc/handlers.go:var stylesheets = []string{"default.css", "custom.css"}
help/en/prevnext.myco:In file `your-wiki/static/custom.css` write this:
help/en/file_structure.myco:** `static/custom.css` is loaded after the main style. If you want to make visual changes to your wiki, this is probably where you should do that.
```

**Comment by bouncepaw** [comment]
It's not ignored, both files get concatenated and served as `/static/style.css`.

For example, look at the stylesheet for my digital garden:
=> https://garden.bouncepaw.com/static/style.css

First, the default stylesheet is listed, then my custom one. They do get applied, see:
=> https://garden.bouncepaw.com

See the responsible code:
=> https://github.com/bouncepaw/mycorrhiza/blob/727280147afd3d2bb6b38582a75cd5643166cff7/misc/handlers.go#L141-L156

Basically, it reads both stylesheets and writes them to the client.

Are your custom styles not applied? Have you refreshed your cache?

**Comment by Lorentz83** [comment]
Apologies, there was a typo in my css and therefore it didn't work, and while debugging I was surprised to not see the file loaded directly by the html. 

Can I suggest to update the documentation? I find pretty confusing that files in static are both served verbatim or used to generate other files. Also, it feels weird that custom js needs to be declared in  `config.ini` while custom css works as long as the file is named correctly

**Comment by bouncepaw** [comment]
These are fair suggestions, thank you
